package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -70,9 +76,40 @@
     with h5py.File("trajectory.h5md", 'r') as f:
          u = mda.Universe("topology.tpr", f)
 
+
 .. Note:: Directly using a `h5py.File` does not work yet.
    See issue `#2884 <https://github.com/MDAnalysis/mdanalysis/issues/2884>`_.
 
+
+Example: Writing an H5MD file
+-----------------------------
+
+To write to an H5MD file from a trajectory loaded with MDAnalysis, do:
+
+.. code-block:: python
+
+    import MDAnalysis as mda
+    u = mda.Universe("topology.tpr", "trajectory.h5md")
+    with mda.Writer("output.h5md", n_atoms=u.trajectory.n_atoms) as W:
+        for ts in u.trajectory:
+            W.write(u)
+
+To write an H5MD file with contiguous datasets, you must specifly the
```

Comment:
Specifly 🪰

Reply:
```suggestion
To write an H5MD file with contiguous datasets, you must specify the
```
I'll just merge it in myself.